examples/exampleApi.go

```diff
@@ -0,0 +1,66 @@
+package examples
+
+import (
+  "fmt"
+  "net/url"
+  "net/http"
+  "github.com/openwhisk/go-whisk/whisk"
+)
+
+func testCreateAction() {
```

Comment:
I suggest we put the test code into separate file, we better obey go best practices